packages/terra-search-field/tests/wdio/search-field-spec.js

```diff
@@ -9,7 +9,7 @@ Terra.describeViewports('Search Field', ['medium'], () => {
     });
 
     it('should enter a search term', () => {
-      $('input').setValue('Lore');
+      $('input').setValue('Lore', { mismatchTolerance: 0.1 });
```

Comment:
Out of curiosity, what is the reason for the mismatchTolerance being set??

Reply:
Those tests were randomly failing due to some of the edges of the focus dotted lines being darker/lighter than the reference

![image](https://github.com/cerner/terra-core/assets/37750902/2a573ee9-11d1-4f6a-a803-5ffd0dbcfa40)

Reply:
That makes sense. Thanks!

Can you add a comment indicating why this was added for documentation purposes? Something like:
`Setting the mismatchTolerance to 0.1 to account for subtle variances in the dotted line.`

(Or whatever wording is more accurate)

Reply:
Good idea! Added comments for the new mismatchTolerances here https://github.com/cerner/terra-core/pull/3834/commits/0ef9818f16cb8cffc519be4a3e2169e11bfbba28